docs/dev/framework/templates.md

```diff
@@ -138,3 +138,37 @@ The output of the `image.html5` template will be:
 <img src="files/images/house.jpg" alt="A small house in England" />
 <small>Photograph by Donna Evans, licensed under Creative Commons</small>
 ```
+
+
+## Template Data
+
+The available template data varies depending on the source of the template. Typically,
+content element templates or element templates of modules will have their complete
+data record available in the template. For example, in any content element template,
+the complete data set of the database record of the element will be available in 
+the template via `$this->…`. The content of some fields might have been changed 
+by the content element or module controller though.
+
+You can inspect the available template data by using either
+
+```php
+<?php $this->dumpTemplateVars() ?>
+```
+
+or
+
+```php
+<?php dump($this) ?>
```

Comment:
```suggestion
<?php dump($this); ?>
```

Reply:
`;` is not necessary in this case. Personally, I prefer it without `;` :)

Reply:
Ok. 👍